src/operator/windowTime.ts

```diff
@@ -96,21 +140,39 @@ interface CloseWindowContext<T> {
 
 interface CloseState<T> {
   subscriber: WindowTimeSubscriber<T>;
-  window: Subject<T>;
+  window: CountedSubject<T>;
   context: CloseWindowContext<T>;
 }
 
+class CountedSubject<T> extends Subject<T> {
+  private numberOfNextedValues: number;
+  constructor() {
+    super();
+    this.numberOfNextedValues = 0;
+  }
+
+  next(value?: T): void {
+    this.numberOfNextedValues++;
+    super.next(value);
+  }
+
+  getNumberOfNextedValues(): number {
```

Comment:
for this one we can simply make it as getter property?

```
private _numberOfNextedValues: number = 0;
public get numberOfNextedValues() {
 return this._numberOfNextedValues;
}
```

This also fits for our further directions to follow private property convention (`_xxx`) for JS users.